website/preprints/tasks.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+import urlparse
+
+import requests
+
+from framework.celery_tasks import app as celery_app
+
+from website import settings
+
+
+@celery_app.task(ignore_results=True)
+def on_preprint_updated(preprint_id):
```

Comment:
In side effects, could you sketch out an integration test (like Runscope) that could verify if there might be any async problems with the celery tasks?